.gitignore

```diff
@@ -11,3 +11,4 @@
 sqlnet.log
 Gemfile.lock
 /spec/sandbox
+.idea
```

Comment:
May we move that to a separate PR (maybe we can also think of other commonly ignored files).